mico-core/src/main/java/io/github/ust/mico/core/service/MicoStatusService.java

```diff
@@ -79,46 +120,59 @@ public MicoApplicationStatusDTO getApplicationStatus(MicoApplication micoApplica
      * @return {@link MicoServiceStatusDTO} which contains status information for a specific {@link MicoService}
      */
     public MicoServiceStatusDTO getServiceStatus(MicoService micoService) {
-        Optional<Deployment> deploymentOptional = null;
+        Optional<Deployment> deploymentOptional = Optional.empty();
+        MicoServiceStatusDTO serviceStatus = new MicoServiceStatusDTO();
         try {
             deploymentOptional = micoKubernetesClient.getDeploymentOfMicoService(micoService);
         } catch (KubernetesResourceException e) {
```

Comment:
Should the execution really continue in this case? I think a service status with error messages would be the best option.